GameServerLib/GameObjects/AttackableUnits/AI/BaseTurret.cs

```diff
@@ -25,7 +24,7 @@ public class BaseTurret : ObjAiBase, IBaseTurret
             uint netId = 0
         ) : base(game, model, new Stats.Stats(), 50, x, y, 1200, netId)
         {
-            ParentNetId = Crc32Algorithm.Compute(Encoding.UTF8.GetBytes(name)) | 0xFF000000;
+            ParentNetId = Force.Crc32.Crc32Algorithm.Compute(Encoding.UTF8.GetBytes(name)) | 0xFF000000;
```

Comment:
not sure why this was inlined

Reply:
VS marked the `using` line as problematic for no reason so I just copied it here. I don't mind changing it back if VS do no problems with that.

Reply:
Yeah we'd rather have one more using directive than a line that's harder to read